code/client/src/core/hooks/render_device.cpp

```diff
@@ -89,7 +89,7 @@ int64_t C_WindowProcHandler__CreateMainWindow(void *_this, SDK::ue::C_Applicatio
     MafiaMP::Game::gGlobals.window = appWin32->m_pWindow;
 
     // Update the main window title asap
-    SetWindowTextA(MafiaMP::Game::gGlobals.window, "Mafia: Advanced Multiplayer Edition");
```

Comment:
This was an internal joke :D Not sure if we should keep it atm

Reply:
Mafia Super Duper Advanced Multiplayer Definitive Edition